containers/Ideas/Idea.js

```diff
@@ -23,6 +23,7 @@ export const Idea = (props) => {
   const [comments, setComments] = useState([]);
   const [contentObjectOfComment, setContentObjectOfComment] = useState({'contentType': idea.content_type, 'pk': idea.pk});
   const [commentLastCommented, setCommentLastCommented] = useState(-1);
+  const [isScrolling, setIsScrolling] = useState(false);
```

Comment:
really cool with the sticky back button and also physical back works!
Just a question regarding the IdeaProject.js. Do we want to have a sticky backbutton there too? Maybe more of a question for @mcastro-lqd .

And if yes, would it be easy to implement this there as well? or shall we see that later?

Reply:
lets leave the back button on IdeaProject out for now and talk about it later